assets/stylesheets/openqa.scss

```diff
@@ -19,7 +19,8 @@ $color-resborder-unk: $color-module-none;
 $color-highlight-parent: #ffe0e0;
 $color-highlight-child: #d0f0ff;
 $color-labels: gray;
-$color-bolt: #ffbf00; /* amber */
+$color-bug: #f2b500; /* amber */
+$color-bug-closed: #6d6d6d; /* gray */
```

Comment:
I don't see the reason why $color-bolt was removed. I recommend to have a distict but not the same color for both bolt as well as bug. Also, currently closed ticket references are showing up in a light grey which looks **acceptable** but IMHO they are even more important to handle than open tickets.